specification/cost-management/resource-manager/Microsoft.CostManagement/preview/2018-12-01-preview/costmanagement.json

```diff
@@ -441,12 +441,57 @@
         "x-ms-examples": {
           "SubscriptionQuery": {
             "$ref": "./examples/SubscriptionQuery.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "#/parameters/apiVersionParameter"
           },
-          "SubscriptionQueryUsage": {
-            "$ref": "./examples/SubscriptionQueryUsage.json"
+          {
+            "$ref": "#/parameters/subscriptionIdParameter"
           },
-          "SubscriptionQueryForecast": {
-            "$ref": "./examples/SubscriptionQueryForecast.json"
+          {
+            "name": "parameters",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/ReportConfigDefinition"
+            },
+            "description": "Parameters supplied to the CreateOrUpdate Report Config operation."
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK. The request has succeeded.",
+            "schema": {
+              "$ref": "#/definitions/QueryResult"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": null
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/providers/Microsoft.CostManagement/Forecast": {
+      "post": {
+        "tags": [
+          "Forecast"
+        ],
+        "operationId": "Forecast_UsageBySubscription",
```

Comment:
>Forecast [](start = 24, length = 8)

should be plural.

Reply:
**Forecast** in this context is a verb, not a noun. This is the same as **Microsoft.CostManagement/query** (verb, not a noun). Same for all the other instances.